sdk/eventgrid/azure-eventgrid/samples/champion_scenarios/cs1_publish_custom_events_to_a_topic.py

```diff
@@ -0,0 +1,19 @@
+from azure.eventgrid import EventGridPublisherClient, EventGridEvent, CloudEvent
+from azure.core.credentials import AzureKeyCredential
+
+topic_hostname = "<YOUR-TOPIC-NAME>.<REGION-NAME>-1.eventgrid.azure.net"
```

Comment:
Typically pluggables are os.environ loaded.  This is low-pri however, use your judgement for if you want to change these now, or if you feel it'll change a lot, add an issue to the backlog to make sure this is dealt with "sooner or later"